cmd/skaffold/app/cmd/parse_config.go

```diff
@@ -193,15 +200,17 @@ func processEachDependency(d latest.ConfigDependency, cfgOpts configOpts, opts c
 		// empty path means configs in the same file
 		path = cfgOpts.file
 	}
-	fi, err := os.Stat(path)
-	if err != nil {
-		if os.IsNotExist(errors.Unwrap(err)) {
-			return nil, fmt.Errorf("could not find skaffold config %s that is referenced as a dependency in config %s", path, cfgOpts.file)
+	if !util.IsURL(path) {
+		fi, err := os.Stat(path)
+		if err != nil {
+			if os.IsNotExist(errors.Unwrap(err)) {
```

Comment:
The unwrap isn't correct: it's possible the `ErrNoExist` is actually a wrapped error.  See the example at
https://golang.org/pkg/os/#IsNotExist

Reply:
This is a weird one. It was existing code so I didn't mess around with it. It's calculating that the `FileNotFound` error from `ReadConfiguration` is wrapped exactly once in `ParseConfig`. So it unwraps exactly once to get the underlying error. 🤦🏽‍♂️ 

I'll convert this to a concrete error type.

Reply:
The [code for `IsNotExist` notes](https://golang.org/src/os/error.go?s=3221:3252#L86):
> New code should use `errors.Is(err, os.ErrNotExist)`

`errors.Is()` will unwrap the error sequentially.

Reply:
oh, got it. Fixed